rpc/common/moc_common_computecommon.proto

```diff
@@ -154,6 +156,23 @@ message VirtualMachineCapabilities {
 	bool IsolatedVmCapable =  1;
 }
 
+message InstanceViewStatus {
```

Comment:
Can we define the variable types as in Azure 
https://github.com/Azure/azure-sdk-for-go/blob/v67.3.0/profiles/2020-09-01/compute/mgmt/compute/enums.go#L648C29-L648C29

https://github.com/Azure/azure-sdk-for-go/blob/v67.3.0/profiles/2020-09-01/compute/mgmt/compute/models.go#L3379